spec/directions/bake_chapter_glossary_spec.rb

```diff
@@ -28,22 +24,12 @@
     )
   end
 
-  let(:metadata) do
-    new_element(
+  it 'works' do
+    metadata = metadata_source.append(child:
```

Comment:
nice use of `append`! can you rename `metadata_source` to `metadata_element` to match the other helpers?  The "source" bit is more unique to the directions code.